src/Inertia.ts

```diff
@@ -163,7 +163,7 @@ export class Inertia implements InertiaContract {
   public redirectBack() {
     const { response } = this.ctx;
 
-    response.status(303).redirect().back();
+    response.status(303).redirect().withQs().back();
```

Comment:
I was actually planning on deprecating this. This was a bit of a hack, since I couldn't figure out a way to use the native redirect method while preserving the correct headers.
I think it makes sense as a convenience method, though. So, please update the the JSDoc (lines 158-162).